### PR TITLE
chore: add the Paging feature

### PR DESCRIPTION
**Pull request by MaximKudriavtsev**

### CODE REVIEW

File: packages/dx-vue-demos/src/demo-sources/grid-paging/none/basic.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import { generateRows } from '../../../demo-data/generator';
+
+export default {
+  data() {
+    return {
+      currentPage: 0,
+      pageSize: 5,
+    };
+  },
+  render() {
```
Comment (kvet) [inline packages/dx-vue-demos/src/demo-sources/grid-paging/none/basic.js]: Write demos using vue templates

File: packages/dx-vue-grid-bootstrap4/src/templates/paging-panel/pagination.js [hunk 2]
```diff
@@ -0,0 +1,203 @@
+import { firstRowOnPage, lastRowOnPage, calculateStartPage } from '@devexpress/dx-grid-core';
+
+const renderPageButtons = (
```
Comment (kvet) [inline packages/dx-vue-grid-bootstrap4/src/templates/paging-panel/pagination.js]: Maybe its better to convert this method to a component?

Comment (MaximKudriavtsev) [reply packages/dx-vue-grid-bootstrap4/src/templates/paging-panel/pagination.js]: No, because Vue can't return a markup array.

Comment (kvet) [reply packages/dx-vue-grid-bootstrap4/src/templates/paging-panel/pagination.js]: You can use functional component. It allows to return multiple root nodes

**Pull request merged**